### PR TITLE
Dag bruck patch modelica

### PR DESCRIPTION
**Pull request by DagBruck**
Revised proposal for Modelica support in SSP. The significant changes are:
- Added the connector kind "acausal", allowing one acausal connector to connected to another,
- Added MIME-type "text/x-modelica" which has an additional attribute "path" which designates the Modelica class.

Compared to the previous proposal which was withdrawn:
- No source attribute for Binary types (replaced by attribute in MIME-type).
- No "rotational" attribute for connectors.

### CODE REVIEW

File: docs/5___ssd.adoc [hunk 1]
```diff
@@ -144,6 +144,27 @@ Boolean / String / Enumeration / Binary |Exactly one of these elements *CAN* be
...
+|RealOutput |ssc:Real |output
+|IntegerInput |ssc:Integer |input
+|IntegerOutput |ssc:Integer |output
+|BooleanInput |ssc:Boolean |input
+|BooleanOutput |ssc:Boolean |output
+|StringInput |ssc:String |input
+|StringOutput |ssc:String |output
+|===
+
+Modelica connectors of more advanced types are mapped the same way as Modelica component models.
```
Comment (pmai) [inline docs/5___ssd.adoc:162]: F2F 2024-06-13: @pmai to provide suggested wording to make the user aware that this very opaque mapping is for now the way to map modelica connectors, but that other, more transparent and detailed mappings might be added in the future, when SSP has better mechanisms, like e.g. terminal support, more specific acausal semantics, etc. without going into details or making any too future looking statements.

**Comment by pmai** [comment]
F2F 2024-06-13: Decision to include modelica-specific parts in the core standard, as the modelica-specific text is
a) fairly small
b) fairly stable (e.g. RealInput/etc. to Real/input mapping is at least as stable if not more so than FMI mappings)
c) Modelica is a core standard of the MA, the same as FMI.

**Comment by pmai** [comment]
The issue whether to still allow/point out the use of a particular URI scheme like 'modelica:' in the source of a Modelica component to locate it and also to indicate that it the component has an implementation and is not abstract shall be discussed in the meeting that discusses merge of this PR once it is finalized.

**Comment by JochenKoehler** [comment]
web meeting 2024-08-13:
Ulrich will do a review on this

File: docs/5___ssd.adoc [hunk 2]
```diff
@@ -530,6 +556,10 @@ Note that source and destination in the following table indicate the resulting d
 Implementations *MUST NOT* specify connections that are not of one of the allowed combinations in the following table.
 Implementations *MUST* ensure that data flow is specified unambiguously, including ensuring that not multiple connections with inbound data flow enter into a connector signifying an input, inout, parameter, or structuralParameter connector of an element, or a local, constant, calculatedParameter, or output connector of an enclosing system.
 
+For connectors of kind unspecified it is ultimately implementation-defined whether and how connections are allowed, as the exact semantics depend on the underlying modeling language.
```
Comment (UKiffmeier) [inline docs/5___ssd.adoc:559]: highlight 'unspecified' as a technical term.

**Review by UKiffmeier — APPROVED** [review]
Review successful. Only one minor comment regarding highlighting of technical terms (line 559)

File: docs/5___ssd.adoc (hunk 2 above)
Comment (pmai) [reply docs/5___ssd.adoc:559]: We do not currently handle this fully consistently throughout the document, I think, however here the names are used without highlight, see non-changed sentence above. So if we wanted to change this, we would need to change this throughout, I think.

**Comment by pmai** [comment]
CCB 2024-09-13: Can be merged as-is after final confirmation by @UKiffmeier that the schema changes are correct.

**Review by UKiffmeier — APPROVED** [review]
Review successful. PR can be completed.

**Pull request merged**